### PR TITLE
Inherit platformFeePercent unless covid-19 initiative

### PR DESCRIPTION
**Pull request by kewitz**
This means resetting the platformFeePercent when moving from a host with fees on top to a host without.

### CODE REVIEW

File: server/models/Collective.js [hunk 1]
```diff
@@ -1800,16 +1800,14 @@ export default function (Sequelize, DataTypes) {
       throw new Error('This host is not open to applications');
     }
 
+    const platformFeePercent = this.tags?.includes('covid-19') ? 0 : hostCollective.platformFeePercent;
```
Comment (znarf) [inline server/models/Collective.js:1803]: What about `EVENT` and `PROJECT`? How do we make sure they inherit their parent?

**Review by znarf — CHANGES_REQUESTED** [review]
I don't get where/how Events and Projects are inheriting the platformFeePercent of their parent.

**Comment by kewitz** [comment]
@znarf I was counting on the whole creation/cascading flow for that.
I'll make it explicit.

**Comment by znarf** [comment]
@kewitz case to support: a Covid-19 collective decide to join another Host and the Host have platformFeePercent=5, the Events of this Collective should have platformFeePercent=0 (from the parent) and not platformFeePercent=5 (from the host).

**Comment by kewitz** [comment]
@znarf yep, in the case the event or project is not tagged with `covid-19`. Just pushed the fix.

**Pull request merged**